src/Raven.Studio/typescript/models/database/index/indexFieldOptions.ts

```diff
@@ -19,11 +19,11 @@ class indexFieldOptions {
 
     static readonly analyzersNamesDictionary: analyzerName[] = [
         { shortName: "Keyword Analyzer", fullName: "KeywordAnalyzer" },
-        { shortName: "LowerCase Keyword Analyzer", fullName: "Raven.Server.Documents.Indexes.Persistence.Lucene.Analyzers.LowerCaseKeywordAnalyzer" },
+        { shortName: "LowerCase Keyword Analyzer", fullName: null }, // default option
         { shortName: "LowerCase Whitespace Analyzer", fullName: "LowerCaseWhitespaceAnalyzer" },
         { shortName: "NGram Analyzer", fullName:"NGramAnalyzer" },
         { shortName: "Simple Analyzer", fullName: "SimpleAnalyzer" },
-        { shortName: "Standard Analyzer", fullName: null }, // default option
```

Comment:
So we were sending `null` even if someone set `Standard Analyzer` what resulted in defaulting to LowerCase analyzer. Nice catch

Reply:
As we talked - this is the default analyzer _only_ for full text search field (`FieldIndexing.Search`). PR will be closed anyway.